### DISCUSSION TITLE
OTEL_RESOURCE_ATTRIBUTES variables

### DISCUSSION THREAD

**Discussion by isma3ilovich** [open-telemetry/opentelemetry-cpp#605] [Q&A]
Hello,

I want to add service name  and version using OTEL_RESOURCE_ATTRIBUTES ,i set the value using export 

`export OTEL_RESOURCE_ATTRIBUTES="service.name=my-service,service.version=v1.0"`

but  it isn’t detected  by SDK. How  i can add these attributes ? 

Thank you.

**Comment by lalitb** [comment]
@isma3ilovich - Thank you for trying out opentelemetry-cpp. The necessary sdk changes to inject resources to exporters is under review (https://github.com/open-telemetry/opentelemetry-cpp/pull/580) for some design discussions.

 Please check the status of this in couple of weeks.
Thanks,
Lalit

**Comment by isma3ilovich** [comment]
@lalitb  Thank your for your reply.

**Comment by lalitb** [comment]
@isma3ilovich  - Just to let you know, this support has been added now (https://github.com/open-telemetry/opentelemetry-cpp/pull/706). You can either use the latest build or the release v0.6.0 (due next week ).

**Reply by isma3ilovich** [reply]
Thanks @lalitb

**Reply by isma3ilovich** [reply]
I confirm that the issue is resolved
